### ISSUE TITLE
Performance: CreateOrder and SendOrder should be the same function

### ISSUE THREAD

**Issue by virus-zip** [orionprotocol/trading-sdk#29]
The CreateOrder and SendOrder should be the same function should be united into one function to increase performance and response times.

**Comment by eugene-mac** [comment]
> The CreateOrder and SendOrder should be the same function should be united into one function to increase performance and response times.

@virus-zip Could you please explain what kind of performance and what response times it should affect?

**Comment by virus-zip** [comment]
Yes, sure. Both are function that the SDK waits for completion. If you want to develop a perfoemance oriented terminal, users should not wait for two completions for signing orders and sending orders. One function could do both things at the same time.
